.changes/api-rs-fs-disk-entry.md

```diff
@@ -0,0 +1,6 @@
+---
+"api": "major"
+"tauri": "major"
+---
+
+**Breaking change** Removed `DiskEtry` struct from `tauri::api::dir`. Use `DirEntry` instead.
```

Comment:
Typo: `DiskEtry` should be `DiskEntry`

Reply:
thank you, I will keep this in mind when I merge all changelogs together